### ISSUE TITLE
Allow Detail's markdown to render table

### ISSUE THREAD

**Issue by LeoMartinDev** [raycast/extensions#2367]
**Describe the feature and the current behavior/state.**

Currently, markdown table does not render as table but as raw text.

**Will this change the current API? How?**

No

**Who will benefit from this feature?**

Probably a lot of people using `Detail` component

**Any Other info.**

<img width="495" alt="CleanShot 2022-07-24 at 13 55 11@2x" src="https://user-images.githubusercontent.com/7238941/180645816-f253539d-713d-4601-9e64-85ad09ac05c2.png">

**Comment by ItsJamesMurray** [comment]
@LeoMartinDev did you ever come up with a creative solution for this?  I'm looking for a fairly simply table implementation like below:

rendered:
|     | Col1 | Col2 |
| --- | --- | --- |
| Row 1 | Lorem | Lorem |
| Row 2 | Lorem | Lorem |
| Row 3 | Lorem | Lorem |

markdown: 
```
|     | Col1 | Col2 |
| --- | --- | --- |
| Row 1 | Lorem | Lorem |
| Row 2 | Lorem | Lorem |
| Row 3 | Lorem | Lorem |
```

This is a pretty large limitation for my extension use case.

**Comment by gardamork** [comment]
According to the [API reference](https://developers.raycast.com/api-reference/user-interface/detail#detail) Raycast uses the CommonMark specification of markdown, which doesn't support tables.

One solution which _should_ work with CommonMark is using [HTML blocks](https://spec.commonmark.org/0.30/#html-blocks) but it seems the Raycast devs have disabled this option.

**Comment by justspacedog** [comment]
I am also working on an extension that would benefit from displaying tables in Markdown. Please add this function.
<img width="862" alt="image" src="https://user-images.githubusercontent.com/8201551/228400539-ed8f847d-708b-4e3c-844e-2d5d388b30d2.png">

**Comment by thomaslombart** [comment]
Fixed in [1.49.3](https://raycastcommunity.slack.com/archives/C02HEMAF2SJ/p1681308909042669).
